lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -208,6 +213,7 @@ class AwsCompileStreamEvents {
                 FunctionName: resolveLambdaTarget(functionName, functionObj),
                 StartingPosition,
                 Enabled,
+                tumblingWindowInSeconds,
```

Comment:
I believe the property has to be capitalized here, so instead of `tumblingWindowInSeconds` it should be `TumblingWindowInSeconds`

Reply:
Oh yes, sorry for the typo.

Reply:
No worries :+1: